retriever/lib/create_scripts.py

```diff
@@ -90,6 +105,7 @@ def create_script_dict(allpacks, path, file, skip_lines):
     allpacks["citation"] = "FILL"
     allpacks["licenses"] = [{"name": "FILL"}]
     allpacks["keywords"] = []
+    allpacks["archived"] = "fill or remove this field if not archived"
```

Comment:
I wouldn't be immediately clear on what this field should be. Is it a logical? If so what about "FILL: 'True' if data file is compressed and 'False' if it is not"?

Reply:
Alternatively we could set it to False which will be the most common case and then include why/how to switch it in the docs.

Reply:
This is only used when the dataset is stored as an archive. It takes on the value of the type of archive. ie, `zip`, `gz`, etc. I think we have this in docs somewhere. We can discus this later for more input. Thanks Ethan.

Reply:
Sounds good. I'll go ahead and merge this in and we can modify that language later if we come up with something better.